src/stories/Blocks/reservation/reservation.scss

```diff
@@ -63,3 +63,17 @@
     margin: 0;
   }
 }
+
+.reservation-form {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  &__content {
+    width: 45%;
+  }
```

Comment:
Please ensure that this will also work on mobile.

To me it seems as if 45% width will become quite narrow on mobile devices.